### PR TITLE
CORE-392 - RNode Test Net Update or Dev/Master Merge

### PR DESCRIPTION
**Pull request by jeremybusk**
This updates rnode version of mini net on dev and master branch merges. Allows test net to updated for observation. This is first phase with more to come like stats verification tests.

https://rchain.atlassian.net/browse/CORE-392

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -60,6 +65,25 @@ matrix:
...
+    sbt_args: -no-colors
+    install:
+      - ./scripts/install_secp.sh
+      - ./scripts/install_sodium.sh
+      - ./scripts/install_bnfc.sh
+      - ./scripts/p2p-test-network.sh
+    addons:
+      apt:
+        sources:
+          - sourceline: 'deb https://dl.bintray.com/sbt/debian /'
```
Comment (nashef) [inline .travis.yml]: What's bintray?

File: scripts/p2p-test-network.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            rnode_cmd="rnode --port 30304 --standalone --name 0f365f1016a54747b384b386b8e85352 > /var/log/rnode.log 2>&1 &"
+        else
+            rnode_cmd="rnode --bootstrap rnode://0f365f1016a54747b384b386b8e85352@10.1.1.2:30304 > /var/log/rnode.log 2>&1 &"
+        fi
+        
+        ssh_tcp_port=$((40000+$i))
+        ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null \
+            -p ${ssh_tcp_port} ${SSH_USERNAME}@repo.rchain.space " 
+                rm rnode_${TRAVIS_BRANCH}_all.deb;
+                wget --quiet https://repo.rchain.space/rnode_${TRAVIS_BRANCH}_all.deb;
```
Comment (nashef) [inline scripts/p2p-test-network.sh]: What is `rchain.space`?

File: scripts/p2p-test-network.sh [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      "${TRAVIS_BRANCH}" = "dev" || \
+      "${TRAVIS_BRANCH}" = "ops-test" ]] \
+    && [[ "${TRAVIS_REPO_SLUG}" = "rchain/rchain" ]] ; then
+    #&& [[ "${TRAVIS_PULL_REQUEST}" = "false" && "${TRAVIS_REPO_SLUG}" = "rchain/rchain" ]] ; then
+
+    echo "Travis branch ${TRAVIS_BRANCH} matched and from repo rchain/rchain. Pushing rnode to Docker repo."
+
+    # Generate rnode debian and rpm packages - push to repo and then to p2p test net
+    sbt -Dsbt.log.noformat=true clean rholang/bnfc:generate node/rpm:packageBin node/debian:packageBin 
+    scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -P 10003 \
```
Comment (nashef) [inline scripts/p2p-test-network.sh]: `StrictHostKeyChecking=no` is basically `AllowManInTheMiddle=yes`. Why not explicitly add the host key in the script?

File: .travis.yml (hunk 1 above)
Comment (jeremybusk) [reply .travis.yml]: It's used for sbt install
https://www.scala-sbt.org/download.html - adds it to sources.
This helps support versions other than in the travis build

**Pull request merged**